package-lock.json

```diff
@@ -0,0 +1,4251 @@
+{
```

Comment:
Should this be in the root, or should it have changed in the `client` folder?

Reply:
It should be fine here, it changed because of adding the mui-datatables pkg